__tests__/app.test.js

```diff
@@ -0,0 +1,45 @@
+const app = require("../app");
+const db = require("../db/data/development-data");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data")
+
+beforeEach(()=>{
+
+return seed(testData)
+}
+)
+
+
+afterAll(() => {
+    if (db.end) db.end();
+  });
+
+
+
+  describe('Get/api/categories', () => {
+    test('Status 200 - should respond with an array of category objects', () => {
+        return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({body})=>{
+            const { categories } = body;
+            expect(categories).toBeInstanceOf(Array);
```

Comment:
Could we please add to this test so that we check the length of the array (the length of the test data) and we also need to check that the properties are there on the categories object (description and slug). Look to use 'objectContaining' for this. 🙂